0.0.8/README.md

```diff
@@ -1,93 +1,170 @@
-```
-If there is no real PMEM device, DRAM can be used to emulate PMEM.
-```
-$ config-pmem.sh --pmem-emulation --mm-capacity 16
-```
+The following instructions assume that you have installed the [OpenShift Container Platform](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/installing/index#installation-process_ocp-installation-overview), customized to your requirements. 
 
-### License Requirement
-You need to obtain a license and provide it to **Memory Machine™**.
+You must have a Kubernetes cluster deployed, and the `kubectl` command-line tool must be configured to communicate with your cluster.
 
-Firstly, label the worker node whose mac address has signed in the license file with label `memory-machine.memverge.com=license-server`.
-```
-$ kubectl label node <memory-machine-license-node> memory-machine.memverge.com=license-server
-```
-Then, create a secret named `memory-machine-license` containing the license under `memverge` namespace.
-```
-$ kubectl create ns memverge
-$ kubectl create secret generic memory-machine-license \
-    -n memverge \
-    --from-file=license=<path-to-your-license-file>
-```
+The [Operator SDK](https://sdk.operatorframework.io/docs/installation/) is a framework used to develop and run Operators. You will use the `operator-sdk` command to run the Memory Machine Operator, so you must have the Operator SDK installed.
 
-### Image Pull Secrets
-Since the **Memory Machine™** images are currently private, you need to create an image pull secret named `memverge-github-dockerconfig` for **memverge namespace** containing github token with read access to the images.
-```
-$ cat <file-contains-github-token> | docker login ghcr.io
-$ kubectl create secret generic memverge-github-dockerconfig \
-    -n memverge \
-    --from-file=.dockerconfigjson=$HOME/.docker/config.json \
-    --type=kubernetes.io/dockerconfigjson
-$ kubectl patch serviceaccount default \
-    -n memverge \
-    -p '{"imagePullSecrets": [{"name": "memverge-github-dockerconfig"}]}'
-```
+## Configuring the Memory Machine Platform
 
-## Operator Installation
-Memory Machine Operator can be installed using `operator-sdk`.
-```
-$ operator-sdk run bundle ghcr.io/memverge/memory-machine-operator-bundle:0.0.8 \
+Before installing Memory Machine Operator, complete the steps described below.
+
+1. Obtain and install a [Memory Machine License](#obtaining-and-installing-a-memory-machine-license).
+2. Configure [PMem](#configuring-pmem).
+
+### Obtaining and Installing a Memory Machine License
+
+Obtain a Memory Machine license from MemVerge. To request a license, send an email to [mvlicense@memverge.com](mailto:mvlicense@memverge.com). Include the MAC address of the node that will contain the Memory Machine pod. This node is the *Memory Machine node*; the license is valid for the entire cluster but is installed only on the Memory Machine node.
+
+Unless otherwise noted, type all commands on the Memory Machine node.
```

Comment:
The commands will be on a client node, which is usually outside the K8S/OpenShift cluster, but has access to the cluster.